src/vivarium_public_health/population/base_population.py

```diff
@@ -235,16 +235,56 @@ def _load_population_structure(self, builder: Builder) -> pd.DataFrame:
             raise ValueError(
                 f"Scaling factor must be a pandas DataFrame. Provided value: {scaling_factor}"
             )
-        scaling_factor = scaling_factor.set_index(
-            [col for col in scaling_factor.columns if col != "value"]
+        formatted_pop_structure, formatted_scaling_factor = self._format_data_inputs(
```

Comment:
If you want to save typing and needing to think about formatted vs non-formatted variables, you can just actually return anything. You didn't make copies in `_format_data_inputs` so you're really just changing the original dataframes anyway and "mirroring" them with the new variables